pkg/util/ovs/OWNERS

```diff
@@ -0,0 +1,15 @@
+reviewers:
+  - pravisankar
+  - dcbw
+  - smarterclayton
```

Comment:
Unless he actually authored/directly owns the code in the package, you can leave him off of reviewers, to leave first level review to the core team that owns it.